### DISCUSSION TITLE
What setting should be the best for mixtral-8x7b-instruct-v0.1.Q5_K_M?

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#4989] [Q&A]
I have 16GB of VRAM and 64GB of RAM? My GPU goes to 99% it take forever to generate prompts. Like 5 minutes to start typing? Once it starts typing it goes quickly. And changes you see I need to make. I might increase or decrease GPU layers or try 4K context? But any ideas are welcomed. Even changing to 4K Context and increasing or decreasing the GPU layers it does not change much. I made it so my VRAM use is only like 90% but more if offloaded to memory. It still takes a long time for every generation to start but only if the conversation is very large I believe. I feel like it worked better for me before and now it's worse for some reason. 

I am downloading 4_M to see if it's any better. 

Any other model loader that might be better? I can't get it working in Ctransformers. 

![Capture](https://github.com/oobabooga/text-generation-webui/assets/38386267/e8f49b12-ed20-462c-a912-ff8efe709940)

**Comment by rlovell3** [comment]
You think you've got problems, I can't even get any model other than a Vicuna to load.  I git pulled snapshot 2023-12-17 as my initial install, and it has been giving me nothing but grief.  Like you, I'm running 65GB RAM, with RTX 4070, (12GB VRAM).

**Reply by viperwasp** [reply]
I lowered context to 4K. The update with llama with Tensor a check box that is off to the right. Check that if it applies to you. It's not in my screenshot as it's a new update. Now this model here at 5_M actually runs half decently. With 12GB your going to need to probably go with 4_M in all honesty but I don't know.
